sql/ushahidi.sql

```diff
@@ -1145,6 +1145,21 @@ CREATE TABLE IF NOT EXISTS `private_message` (
 -- --------------------------------------------------------
 
 /**
+ * Table structure for table `faq`
+ *
+ */
+
+CREATE TABLE IF NOT EXISTS `faq` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
```

Comment:
where does this number 11 come from ?  was is stated in the requirements/specs that id will be this long ?